audio-worklet/lib/param-controller.js

```diff
@@ -16,37 +16,40 @@
 class ParamController {
   /**
    * Event handler and layout for audio parameters with numeric input values
-   * @param {String} parentId id for parent element.
+   * @param {String} parentId ID for parent element.
    * @param {Function} onChangeCallback Callback to trigger on input.
-   * @param {Object} options
-   * @param {String} options.name The name of the parameter.
+   * @param {Object} options Parameters with default values if unspecified.
+   * @param {String} options.name The name of the parameter. Default is
+   *                              'Parameter'.
    * @param {String} options.id A variable name associated with the parameter.
+   *                            Defaults to the value of |name|.
    * @param {String} options.type The type of input (only support for range).
-   * @param {Number} options.min The minimum possible value.
-   * @param {Number} options.max The maximum possible value.
+   * @param {Number} options.min The minimum possible value. Default is 0.
+   * @param {Number} options.max The maximum possible value. Default is 1.
    * @param {Number} options.step The parameter's increment value.
-   * @param {Number} options.default The default value of the parameter.
+   *                              Default is 0.1.
+   * @param {Number} options.default The default value of the parameter. Default
+   *                                 is 0.
    */
   constructor(parentId, onChangeCallback, options) {
-    if (options == null) options = {};
+    if (typeof options === 'undefined') options = {};
     
     this.controller_ = document.createElement('input');
     this.name_ = options.name || 'Parameter';
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
 
-    if (options.max == null)
+    if (typeof options.max === 'undefined')
```

Comment:
> let x = new ParamController(containerId, this.setGain.bind(this), {});

From my perspective, the last argument here is redundant. If you don't provide the third arg, it will be `undefined`. If you provide anything, it will be something of true. So technically what you have to do is to check if the third arg is `typeof arg === object`. If so, try to parse the properties. If not, simply ignore the argument.